package.json

```diff
@@ -18,6 +18,8 @@
   },
   "dependencies": {
     "@ffmpeg-installer/ffmpeg": "^1.1.0",
+    "dayjs": "^1.11.2",
```

Comment:
I see you're implementing Day.js. \
Please add issue #60 to the linked issues if this includes everything you wanted to include in that issue.

Reply:
No, these aren't the changes that I wanted to do, the changes are actually pretty huge and will need to be done on a separate PR.